React/Views/RCTScrollView.m

```diff
@@ -410,16 +411,24 @@ - (void)setRemoveClippedSubviews:(__unused BOOL)removeClippedSubviews
 
 - (void)insertReactSubview:(UIView *)view atIndex:(__unused NSInteger)atIndex
 {
-  RCTAssert(_contentView == nil, @"RCTScrollView may only contain a single subview");
-  _contentView = view;
-  [_scrollView addSubview:view];
+  if ([view isKindOfClass:[RCTRefreshControl class]]) {
+    _scrollView.refreshControl = (RCTRefreshControl*)view;
```

Comment:
You'll also need to modify the reactSubviews function to return _scrollView.refreshControl in addition to _contentView, otherwise I suspect bad things would happen if you ever tried to remove the refresh control from an existing scrollview.